docs/development_suite/overview-dev-suite.md

```diff
@@ -4,28 +4,47 @@ title: Console Overview
 sidebar_label: Console Overview
 sidebar_order: 0
 ---
-Welcome to the Mia-Platform Console Overview page!
+This page will give you a brief overview of **Mia-Platform Console**, explain **why it's important**, highlight the **advantages of adopting this internal platform solution** and provide a **brief description of its architecture**. 
 
-![Mia-Platform Console Overview](img/Mia-Platform-console-overview.png)
+## Product overview 
 
-In an omnichannel and evolutionary market, successful enterprises are the ones with the capabilities to operate and innovate like tech companies:
+In an evolving, omnichannel market, successful enterprises must operate and innovate like tech companies by:
 
-* **Speed**: Being able to build and launch new digital products with great speed;
-* **Evolvability**: Continuously looking at the ongoing market changes as opportunities;
-* **Scalability**: Being flexible and ready to answer rapidly to demand expansions or contractions;
-* **Openness**: Being able to integrate with tools, partner channels, platforms and ecosystems.
+* Building and launching digital products **quickly**.
+* Viewing market changes as **opportunities**.
+* Being **flexible** to rapidly respond to demand fluctuations.
+* **Integrating** with various tools, partners, and ecosystems.
 
-Modern companies build their digital products, data and logics through a Digital Platform.  
-The tool to reach this goal is the **Internal Developer Portal** which will help you to industrialize and govern cloud-native development and operations. It allows to:
+Modern companies achieve this through an Internal Developer Platform (IDP). The tool to accomplish this is **Mia-Platform Console**, an Internal Developer Platform builder which aids in industrializing, managing and simplifying cloud-native development and operations.
```

Comment:
```suggestion
Modern companies achieve this through an Internal Developer Platform (IDP). The tool to accomplish this is **Mia-Platform Console**, an Internal Developer Platform builder which helps you industrialize and govern cloud-native development and operations.
```

Reply:
Here, I would also include a list of capabilities that exemplify the product, such as connecting your clusters (even across multiple cloud providers), designing your application from services to APIs, managing security, orchestrating workflows, integrating data from third parties, etc. The list is purely for explanatory purposes.

Reply:
Ok, here is my proposal:

Modern companies achieve this through an Internal Developer Platform (IDP). **Mia-Platform Console** is the tool that best fits this need, enabling you to quickly and easily build your own fully customized IDP. Mia-Platform Console also allow you to industrialize, manage, and simplify cloud-native development and operations such as connecting your cluster (even across multiple cloud providers), designing your application from services to APIs, orchestrating workflows and so on.